locales/en/breaches.ftl

```diff
@@ -7,6 +7,9 @@
 # $email-select is an interactive <select> element displaying the current email address
 breach-heading-email = Data breaches for { $email-select }
 
+# $count is the number of emails a user has added out of $total allowed
+emails-monitored = { $count } of { $total } emails monitored
```

Comment:
We should make this a plural, even if English might not need it (it will trigger the plural UI in Pontoon)

```
emails-monitored = 
  {
    $total ->
      [one] { $count } of { $total } email monitored
     *[other] { $count } of { $total } emails monitored
  }
```

Reply:
updated to your code.  Didn't know Pontoon could do that – thanks!